typings/index.d.ts

```diff
@@ -170,8 +160,10 @@ declare module 'discord-giveaways' {
         private ensureEndTimeout(): void;
         private checkWinnerEntry(user: User): Promise<boolean>;
         public checkBonusEntries(user: User): Promise<number>;
-        public fillInString(string: string): string | null;
-        public fillInEmbed(embed: MessageEmbed | MessageEmbedOptions): MessageEmbed | null;
+        public fillInString(string: undefined | null): null;
```

Comment:
Maybe using a conditional type would be better, cause this doesn't include number, boolean etc. right?
And google didn't really tell me a way to specify "!string".

Reply:
I've changed it, as u said. 

The output is now `string | null` it's because if you enter a data which is `string | undefined`, then it should return `string | null`. But if you enter a data which is string, then it'll return string only.